### PR TITLE
fiximg

### PR DESCRIPTION
**Pull request by hieupc-0248**
## Related Tickets:
- [#69609](https://edu-redmine.sun-asterisk.vn/issues/69609)

## Notes Impacted Areas:
- app/Http/Controllers/BookController.php
- resources/lang/vi.json
- resources/js/dashboard.js
- resources/views/books

## Attachments:
- 
image

### CODE REVIEW

**Comment by hieupc-0248** [comment]
ready

**Comment by anhcnn-0378** [comment]
Oke

File: resources/js/dashboard.js [hunk 1]
```diff
@@ -20,10 +23,10 @@ document.addEventListener('DOMContentLoaded', function() {
                     </button>
                 </div>
             `;
-
+    
             bookListElement.appendChild(bookElement);
         });
-    }
+    }
```
Comment (ghost) [inline resources/js/dashboard.js]: ???

**Comment by hieupc-0248** [comment]
ready

File: resources/js/dashboard.js [hunk 2]
```diff
@@ -5,13 +5,16 @@ document.addEventListener('DOMContentLoaded', function() {
 
     function updateBookList(books) {
         bookListElement.innerHTML = '';
-
+
```
Comment (ghost) [inline resources/js/dashboard.js]: ?

File: resources/js/dashboard.js [hunk 3]
```diff
@@ -20,7 +23,7 @@ document.addEventListener('DOMContentLoaded', function() {
                     </button>
                 </div>
             `;
-
+
```
Comment (ghost) [inline resources/js/dashboard.js]: ?

File: resources/views/books/show.blade.php [hunk 4]
```diff
@@ -8,12 +8,18 @@
...
+                    $avtMedia = $book->medias->where('type', '=', config('app.avatar_media_type'))->first();
+                    $imageAvtUrl = $avtMedia ? asset(Storage::url($avtMedia->link)) : asset('storage/default.jpg');
+                @endphp
+                <img id="large-image" class="w-80 h-80" src="{{ $imageAvtUrl }}" alt="">
             </div>
             <div class="flex justify-center h-32 items-center">
                 <div class="py-2 mt-4 w-10/12 bg-gray-200 flex justify-center">
                     @foreach ($book->medias as $media)
-                    <img class="px-1 w-24 h-24" src="{{ asset($media->link) }}" alt="">
+                        @if($media->type == config('app.normal_media_type'))
```
Comment (ghost) [inline resources/views/books/show.blade.php]:
```suggestion
                        @if ($media->type == config('app.normal_media_type'))
```

**Comment by hieupc-0248** [comment]
ready

**Pull request merged**